src/Libraries/DSIronPython/IronPythonEval.csproj

```diff
@@ -1,79 +1,79 @@
+	  </ImportGroup>
+	  <PropertyGroup>
+	    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+	    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+	    <ProjectGuid>{9EEF4F42-6B3B-4358-9A8A-C2701539A822}</ProjectGuid>
+	    <OutputType>Library</OutputType>
+	    <AppDesignerFolder>Properties</AppDesignerFolder>
+	    <RootNamespace>DSIronPython</RootNamespace>
+	    <AssemblyName>DSIronPython</AssemblyName>
+	    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+	    <FileAlignment>512</FileAlignment>
+	    <TargetFrameworkProfile />
+	  </PropertyGroup>
+	  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+	    <DebugSymbols>true</DebugSymbols>
+	    <DebugType>full</DebugType>
+	    <Optimize>false</Optimize>
+	    <OutputPath>$(OutputPath)</OutputPath>
+	    <DefineConstants>DEBUG;TRACE</DefineConstants>
+	    <ErrorReport>prompt</ErrorReport>
+	    <WarningLevel>4</WarningLevel>
+	  </PropertyGroup>
+	  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+	    <DebugType>pdbonly</DebugType>
+	    <Optimize>true</Optimize>
+	    <OutputPath>$(OutputPath)</OutputPath>
+	    <DefineConstants>TRACE</DefineConstants>
+	    <ErrorReport>prompt</ErrorReport>
+	    <WarningLevel>4</WarningLevel>
+	  </PropertyGroup>
+	  <ItemGroup>
+	    <Reference Include="IronPython">
+	      <HintPath>..\..\..\extern\IronPython\IronPython.dll</HintPath>
+	    </Reference>
+	    <Reference Include="Microsoft.Scripting">
+	      <HintPath>..\..\..\extern\IronPython\Microsoft.Scripting.dll</HintPath>
+	      <Private>True</Private>
+	    </Reference>
+	    <Reference Include="ProtoInterface, Version=0.0.0.0, Culture=neutral, processorArchitecture=AMD64">
+	      <SpecificVersion>False</SpecificVersion>
+	      <HintPath>..\..\..\extern\ProtoGeometry\ProtoInterface.dll</HintPath>
+	      <Private>False</Private>
+	    </Reference>
+	    <Reference Include="System" />
+	    <Reference Include="System.Core" />
+	    <Reference Include="System.Xml.Linq" />
+	    <Reference Include="System.Data.DataSetExtensions" />
+	    <Reference Include="Microsoft.CSharp" />
+	    <Reference Include="System.Data" />
+	    <Reference Include="System.Xml" />
+	  </ItemGroup>
+	  <ItemGroup>
+	    <Compile Include="..\..\AssemblySharedInfoGenerator\AssemblySharedInfo.cs">
+	      <Link>AssemblySharedInfo.cs</Link>
+	    </Compile>
+	    <Compile Include="IronPythonEvaluator.cs" />
+	    <Compile Include="Properties\AssemblyInfo.cs" />
+	  </ItemGroup>
+	  <ItemGroup>
+	    <Content Include="DSIronPython_DynamoCustomization.xml" />
+	  </ItemGroup>
+	  <ItemGroup>
+	    <ProjectReference Include="..\..\DynamoUtilities\DynamoUtilities.csproj">
+	      <Project>{b5f435cb-0d8a-40b1-a4f7-5ecb3ce792a9}</Project>
+	      <Name>DynamoUtilities</Name>
+	    </ProjectReference>
+	  </ItemGroup>
+	  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
```

Comment:
I don't know how I managed to introduce these new lines.  The only actual changes are below.